tox.ini

```diff
@@ -24,6 +22,7 @@ deps =
     coverage: coverage-enable-subprocess
 
     pytest54: pytest>=5.4,<5.5
+    psycopg28: psycopg2-binary==2.8.6
```

Comment:
Why is this needed?

Reply:
Seems we don't need it now. I can't remember why I added this line.
Removed!